### DISCUSSION TITLE
Esphome yaml package

### DISCUSSION THREAD

**Discussion by szupi-ipuzs** [andrew-codechimp/HA-Battery-Notes#1889] [Show and Tell]
As many of the esphome users create their own devices (or reprogram existing), they often might want to see similar battery info for these devices as your Battery Notes shows. I sure did :)
That's why I created [this esphome package](https://github.com/szupi-ipuzs/esphome-packages/tree/main#battery_notesyaml). I hope you don't mind, I have copied your ideas for HA entities.

![obraz](https://github.com/user-attachments/assets/21bda5b9-d25c-447c-85f7-d797e71d3838)

**Comment by andrew-codechimp** [comment]
That's really interesting using a package like that in ESP Home, I've not done that before.
Battery Notes can be added to ESP devices, you'd just set them up manually within Battery Notes but it's cool having it on device.
No worries at all about taking inspiration from this project, I've learnt bits from dozens of projects to get this created so it's nice to know it's helping others as well.
